projects/plugins/automattic-for-agencies-client/src/js/components/connection-card/index.jsx

```diff
@@ -1,61 +1,333 @@
+						'automattic-for-agencies-client'
+					) }
+				</li>
+				<li>
+					<CheckIcon />
+					{ __( 'Receive instant downtime alerts', 'automattic-for-agencies-client' ) }
+				</li>
+				<li>
+					<CheckIcon />
+					{ __( 'And more', 'automattic-for-agencies-client' ) }
+				</li>
+			</ul>
+			<div className={ styles[ 'site-connection' ] }>
+				<div className={ styles[ 'terms-of-service' ] }>
+					{ createInterpolateElement(
+						__(
+							'By clicking the <strong>connect this site</strong> button, you agree to our <tosLink>Terms of Service</tosLink> and to <shareDetailsLink>share details</shareDetailsLink> with WordPress.com.',
+							'automattic-for-agencies-client'
+						),
+						{
+							strong: <strong />,
+							tosLink: (
+								<a
+									className={ styles[ 'terms-of-service__link' ] }
+									href={ getRedirectUrl( 'wpcom-tos' ) }
+									rel="noopener noreferrer"
+									target="_blank"
+								/>
+							),
+							shareDetailsLink: (
+								<button
+									className="components-button is-link"
+									onClick={ onShowSharingDetailsClick }
+								/>
+							),
+						}
+					) }
+				</div>
+				<div className={ styles[ 'connect-button-wrapper' ] }>
+					<ConnectButton
+						connectLabel={ __( 'Connect this site', 'automattic-for-agencies-client' ) }
+						apiRoot={ apiRoot }
+						apiNonce={ apiNonce }
+						registrationNonce={ registrationNonce }
+						from="automattic-for-agencies-client"
+						redirectUri="admin.php?page=automattic-for-agencies-client"
+					/>
+				</div>
+			</div>
+		</>
+	);
+}
+
+/**
+ * Detail Sharing Content
+ * The copy for the "share details" information linked from the TOS.
+ *
+ * @param {object}   props                             - Component props
+ * @param {Function} props.onCloseSharingDetailsClick  - Callback to close the sharing details.
+ *
+ * @returns {React.Component} The `DetailSharingContent` component.
+ */
+function DetailSharingContent( { onCloseSharingDetailsClick } ) {
+	return (
+		<>
+			<div>
+				<button
+					className="components-button components-button--back is-link"
+					onClick={ onCloseSharingDetailsClick }
+				>
+					<LeftArrow />
+					{ __( 'Go back to site connection', 'automattic-for-agencies-client' ) }
+				</button>
+			</div>
+			<div className={ styles.card__prose }>
+				<h2>What data is synced from your site</h2>
+				<p>
+					Your privacy matters to us. We sync only the necessary data required to provide our
+					Automattic for Agencies portal experiences.
+				</p>
+			</div>
+			<ul className={ styles.checklist }>
+				<li>
+					<CheckIcon />
+					{ createInterpolateElement(
+						__(
+							'<strong>Options:</strong> to sync site options data.',
+							'automattic-for-agencies-client'
+						),
+						{ strong: <strong /> }
+					) }
+				</li>
+				<li>
+					<CheckIcon />
+					{ createInterpolateElement(
+						__(
+							'<strong>Callables:</strong> to sync for callables.',
```

Comment:
Is this temporary copy? I'm asking because I don't know if many site owners will know what "Callables" refers to, for example.

Reply:
I'm not sure. On one hand, I think we should be as exhaustive as possible to be fully transparent. On the other hand, this may be too technical for this use-case. 

@Automattic/jetpack-vulcan I'd be curious to have your take on this. When explaining Sync, how should we explain each module, and should we even explain each module, or should we say that we're synchronizing some of your site's options?

Reply:
When it comes to Sync, we generally try to avoid any references to it on its own, because they can be confusing to our end users and instead tend to mention the corresponding features Sync powers.

I saw @rcanepa will open a new issue for copy updates so I would like to offer the following suggestions:
- What data is **synced** from your site -> What data is **shared** from your site with WordPress.com (we are more consistent with the ToS agreement copy too)
- We **sync** only the necessary data -> We **collect** only the necessary data
- Options/Callables/Constants -> **Options**: We collect certain site options (and constants) that help us identify changes to the features that we power
- Full_Sync_Immediately -> We can drop this. Full Sync is just an alternate way Sync works to sync it's configured content
- Meta -> Maybe we can drop this too. If we mention for example that we share the site's users and any related changes I think we cover dev related contexts, such as: Users and Usermeta
- Stats -> Maybe we could mention what heartbeat data means and why we need them :)
- Data we don't Sync -> I think this might be redundant since we explicitly mention what we share with WP.com 

In general it would be nice to explain not only what we share with WP.com but also why. For example: "**Plugins**: We share plugin related info in order to enable remote plugin management"

Reply:
This is great, @fgiannar. I'll review your suggestions with @simonktnga8c to create a better copy for this.

FWIW, here is the issue I created to track this work (https://github.com/Automattic/automattic-for-agencies-dev/issues/218).